schemas/2015-08-19/Microsoft.Search.json

```diff
@@ -0,0 +1,139 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-08-19/Microsoft.Search.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Search",
+  "description": "Microsoft Search Resource Types",
+  "resourceDefinitions": {
+    "searchServices": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Search/searchServices"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2015-08-19"
+          ]
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/SearchServiceProperties"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Properties of the Search service."
+        },
+        "location": {
+          "oneOf": [
+            {
+              "type": "string"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The geographic location of the resource. This must be one of the supported and registered Azure Geo Regions (for example, West US, East US, Southeast Asia, and so forth)."
```

Comment:
I don't think we document that info anywhere since it changes over time, but it is possible to obtain it programmatically. I'll update the description with a link to info on how to do this.